### PR TITLE
Add recipe for mrcfile

### PR DESCRIPTION
**Pull request by bkpoon**

### CODE REVIEW

**Comment by bkpoon** [comment]
@conda-forge/staged-recipes Please review, thanks!

File: recipes/mrcfile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 689bb3a094500d0025abb0bcb24b75558dde3f68fe659c8d610b15aa6180dbc9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```
Comment (scopatz) [inline recipes/mrcfile/meta.yaml]: Can this be `noarch: python`

File: recipes/mrcfile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    with no dependencies on any compiled libraries except numpy. You can
+    use it interactively to inspect files, correct headers and so on, or
+    in scripts and larger software packages to provide basic MRC file
+    I/O functions.
+  doc_url: https://mrcfile.readthedocs.io
+  dev_url: https://github.com/ccpem/mrcfile
+
+extra:
+  recipe-maintainers:
+    - colinpalmer
```
Comment (scopatz) [inline recipes/mrcfile/meta.yaml]: @colinpalmer are you OK being a maintainer for this feedstock?

File: recipes/mrcfile/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/mrcfile/meta.yaml]: Ahh ok.  Super annoying

File: recipes/mrcfile/meta.yaml (hunk 2 above)
Comment (colinpalmer) [reply recipes/mrcfile/meta.yaml]: Yes, that's fine with me.

**Comment by bkpoon** [comment]
Thanks!

**Pull request merged**